core/pkg/sync/grpc/grpc_sync.go

```diff
@@ -169,15 +169,15 @@ func (g *Sync) connectWithRetry(
 			return nil, false
 		}
 
-		g.Logger.Warn(fmt.Sprintf("connection re-establishment attempt in-progress for grpc target: %s", g.URI))
+		g.Logger.Warn(fmt.Sprintf("connection re-establishment attempt in-progress for grpc target: %s and authority: %s", g.URI, g.ServAuthority))
```

Comment:
This will log empty authority in case of custom authority is not provided

Reply:
But since it's our container, we will always use authority